src/pathfora.js

```diff
@@ -138,7 +137,7 @@
 
   // NOTE HTML templates
   // FUTURE Move to separate files and concat
-  var templates = {{templates}};
+  var templates = '{{templates}}';
```

Comment:
pretty sure this will break the templates.

the linter will only work on the actual compiled file. anything uses the double brackets will throw an error. probably a way to ignore those in the config but this also skips the actual js that comes in for the templates which should follow the same patterns.